dataland-dummy-qa-service/DockerfileTest

```diff
@@ -0,0 +1,4 @@
+ARG DATALAND_DUMMY_QA_SERVICE_BASE_VERSION
+FROM ghcr.io/d-fine/dataland/dataland_dummy_qa_service_base:${DATALAND_DUMMY_QA_SERVICE_BASE_VERSION}
+
+CMD ./gradlew :dataland-dummy-qa-service:bootRun --args='--spring.profiles.active=ci' --no-daemon --stacktrace
```

Comment:
Use the jacoco agent directly - see the other spring-services as example

Reply:
Done.

Reply:
Done